crates/plugins/mediator-coordination/src/web/handler.rs

```diff
@@ -54,7 +58,7 @@ async fn process_response_from_delegate_handler(
 
     // Pack response message
     let packed_message = match web::midlw::pack_response_message(
-        &plain_response_message,
+        &plain_response_message.unwrap(),
         &state.did_resolver,
```

Comment:
Your forward delegate handler will inevitably fail because it returns a None response, which cannot be unwrapped. What you must do here is to conditionally pack the response message if there is Some, otherwise reply with an empty string.